timeseries/tests/unittests/test_ts_dataset.py

```diff
@@ -251,7 +235,9 @@ def test_subsequence(start_timestamp, end_timestamp, item_ids, datetimes, target
         (["2020-01-01 00:00:00", "2020-01-01 01:00:00", "2020-01-01 02:00:00"], "H"),
     ],
 )
-def test_when_dataset_constructed_from_dataframe_without_freq_then_freq_is_inferred(timestamps, expected_freq):
```

Comment:
It seems that you have `black` running with a different value of `line-length` (not 119 as in `pyproject.toml`), which generates these redundant formatting changes

Reply:
👍 blacking this file again.